BeerOverflowWindowsApp/BeerPanel.cs

```diff
@@ -77,7 +77,8 @@ private void BeerPanel_MouseMove(object sender, MouseEventArgs e)
                 if (marker.Y + marker.Height / 2 > bottom.Y)
                     marker.Y = bottom.Y - marker.Height / 2;
                 Rating = (bottom.Y - marker.Y - marker.Height / 2) / (height / 100) + "";
```

Comment:
Perhaps I would suggest to take into consideration another way to meter the amount of poured beer, like professor said, it should have not percentage, but some kind of point rating system, like 1 is "i had a fight with bartender" and 9 "spilled my beer when tried to sip" to 10 "BEER OVERFLOW". What do others think?

Reply:
Will do that